UnknownHostException

When I try to run the [first example](https://tumblr.github.io/colossus/docs/quickstart/#build-a-hello-world-service), it just crashes and outputs this:

```
(run-main-2) java.net.UnknownHostException: renzo: renzo: Name or service not known
java.net.UnknownHostException: renzo: renzo: Name or service not known
        at java.net.InetAddress.getLocalHost(InetAddress.java:1505)
        at colossus.metrics.MetricSystem.<init>(MetricSystem.scala:155)
        at colossus.metrics.MetricSystem$.apply(MetricSystem.scala:192)
        at colossus.metrics.MetricSystem$.apply(MetricSystem.scala:202)
        at colossus.IOSystem$$anonfun$2.apply(IOSystem.scala:53)
        at colossus.IOSystem$$anonfun$2.apply(IOSystem.scala:53)
        at scala.Option.getOrElse(Option.scala:121)
        at colossus.IOSystem$.apply(IOSystem.scala:53)
        at colossus.IOSystem$.apply(IOSystem.scala:30)
        at explorer.Main$.delayedEndpoint$explorer$Main$1(explorer.scala:21)
        at explorer.Main$delayedInit$body.apply(explorer.scala:18)
        at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
        at scala.App$class.main(App.scala:76)
        at explorer.Main$.main(explorer.scala:18)
        at explorer.Main.main(explorer.scala)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
Caused by: java.net.UnknownHostException: renzo: Name or service not known
        at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
        at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:928)
        at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1323)
        at java.net.InetAddress.getLocalHost(InetAddress.java:1500)
        at colossus.metrics.MetricSystem.<init>(MetricSystem.scala:155)
        at colossus.metrics.MetricSystem$.apply(MetricSystem.scala:192)
        at colossus.metrics.MetricSystem$.apply(MetricSystem.scala:202)
        at colossus.IOSystem$$anonfun$2.apply(IOSystem.scala:53)
        at colossus.IOSystem$$anonfun$2.apply(IOSystem.scala:53)
        at scala.Option.getOrElse(Option.scala:121)
        at colossus.IOSystem$.apply(IOSystem.scala:53)
        at colossus.IOSystem$.apply(IOSystem.scala:30)
        at explorer.Main$.delayedEndpoint$explorer$Main$1(explorer.scala:21)
        at explorer.Main$delayedInit$body.apply(explorer.scala:18)
        at scala.Function0$class.apply$mcV$sp(Function0.scala:34)
        at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.App$$anonfun$main$1.apply(App.scala:76)
        at scala.collection.immutable.List.foreach(List.scala:381)
        at scala.collection.generic.TraversableForwarder$class.foreach(TraversableForwarder.scala:35)
        at scala.App$class.main(App.scala:76)
        at explorer.Main$.main(explorer.scala:18)
        at explorer.Main.main(explorer.scala)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
```

`renzo` is my hostname, and it does not resolve to an IP address.

--- a/colossus-metrics/src/main/scala/colossus/metrics/MetricSystem.scala
+++ b/colossus-metrics/src/main/scala/colossus/metrics/MetricSystem.scala
@@ -152,7 +152,12 @@ class MetricSystem private[metrics] (val config: MetricSystemConfig)(implicit sy
 
   val namespace: MetricAddress = "/"
 
-  private val localHostname = java.net.InetAddress.getLocalHost.getHostName
+  private lazy val localHostname = try  {
+    java.net.InetAddress.getLocalHost.getHostName
+  } catch {
+    case e: java.net.UnknownHostException => "unknown_host"
+  }
+
   val tags: TagMap = Map("host" -> localHostname)
 
   protected val collection = new Collection(config.collectorConfig)

